server/util/grpcutil/grpcutil.go

```diff
@@ -0,0 +1,24 @@
+package grpcutil
```

Comment:
we already have a `rpcutil` if you want to stuff this in there

Reply:
Ah, duh, done